scripts/run_carp_tests.sh

```diff
@@ -43,7 +43,6 @@ echo "Compile-only examples"
 compileOnlyExamples="\
                    examples/mutual_recursion.carp \
                    examples/guessing_game.carp \
-                   examples/unicode.carp \
```

Comment:
We should restore this one if #994 is applied.

Reply:
Awesome, I'll put it back!